dev/release/setup-AlmaLinux-Rocky-Linux.sh

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# A script to install dependencies required for release
+# verification on AlmaLinux 8 and Rocky Linux 8
+
+if [ $# -lt 2 ]; then
+  echo "Usage: $0 VERSION rc"
+  echo "       $0 VERSION staging-rc"
+  echo "       $0 VERSION release"
+  echo "       $0 VERSION staging-release"
+  echo "       $0 VERSION local"
+  echo " e.g.: $0 0.13.0 3                 # Verify 0.13.0 3"
+  echo " e.g.: $0 0.13.0 staging-rc        # Verify 0.13.0 RC on staging"
+  echo " e.g.: $0 0.13.0 release           # Verify 0.13.0"
+  echo " e.g.: $0 0.13.0 staging-release   # Verify 0.13.0 on staging"
+  echo " e.g.: $0 0.13.0-dev20210203 local # Verify 0.13.0-dev20210203 on local"
+  exit 1
+fi
+
+VERSION="$1"
+TYPE="$2"
+
+sudo dnf -y update
+sudo dnf clean all
+sudo dnf -y install dnf-plugins-core
+sudo dnf config-manager --set-enabled powertools
+sudo dnf update
+sudo dnf -y install \
+  https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+sudo dnf -y install ninja-build libcurl-devel python3-pip cmake git 
+sudo dnf -y ncurses-devel gobject-introspection-devel libffi-devel
+sudo dnf -y install openssl-devel maven java-1.8.0-openjdk-devel
+sudo dnf install -y yum-utils readline-devel gdbm-devel ruby-devel
+sudo dnf -y groupinstall "Development Tools"
+sudo dnf -y module enable llvm-toolset
+sudo dnf -y module install llvm-toolset
```

Comment:
Are they needed?

`llvm-toolset` is included in appstream too.

Reply:
ok, can install directly.